### PR TITLE
fix: init controller-runtime logger

### PR DESCRIPTION
**Pull request by CeerDecy**
#### What this PR does / why we need it:
init controller-runtime logger
reference [Github issue](https://github.com/rook/rook/issues/12434)

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/all?id=641448&iterationID=-1&type=BUG)

#### Specified Reviewers:

/assign @iutx 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |       init controller-runtime logger       |
| 🇨🇳 中文    |      修复controller-runtime报错        |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by iutx — DISMISSED** [review]
LGTM @sfwn

File: go.mod [hunk 1]
```diff
@@ -474,14 +480,10 @@ require (
...
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
-	filippo.io/edwards25519 v1.1.0 // indirect
-	github.com/expr-lang/expr v1.16.9 // indirect
-	github.com/gabriel-vasile/mimetype v1.4.3 // indirect
-	github.com/kedacore/keda/v2 v2.15.1
-	gopkg.in/evanphx/json-patch.v4 v4.12.0 // indirect
-	knative.dev/pkg v0.0.0-20240805063731-c88d5dad9653 // indirect
 )
 
+require github.com/sykesm/zap-logfmt v0.0.4 // indirect
```
Comment (sfwn) [inline go.mod:485]: move into upper require block

**Review by sfwn — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (CeerDecy) [reply go.mod]: done

**Review by sfwn — APPROVED** [review]

**Pull request merged**